mlflow/utils/_unity_catalog_utils.py

```diff
@@ -374,3 +393,57 @@ def get_full_name_from_sc(name, spark) -> str:
         return f"{catalog}.{name}"
     schema = spark.sql(_ACTIVE_SCHEMA_QUERY).collect()[0]["schema"]
     return f"{catalog}.{schema}.{name}"
+
+
+def is_databricks_sdk_models_artifact_repository_enabled(host_creds):
+    # Return early if the environment variable is set to use the SDK models artifact repository
+    if MLFLOW_USE_DATABRICKS_SDK_MODEL_ARTIFACTS_REPO_FOR_UC.get():
+        return True
+
+    endpoint, method = _METHOD_TO_INFO[IsDatabricksSdkModelsArtifactRepositoryEnabledRequest]
+    req_body = message_to_json(IsDatabricksSdkModelsArtifactRepositoryEnabledRequest())
+    response_proto = IsDatabricksSdkModelsArtifactRepositoryEnabledResponse()
+
+    # Default fallback value
+    is_enabled = False
+    try:
+        resp = call_endpoint(
+            host_creds=host_creds,
+            endpoint=endpoint,
+            method=method,
+            json_body=req_body,
+            response_proto=response_proto,
+        )
+        is_enabled = resp.is_databricks_sdk_models_artifact_repository_enabled
+    except Exception as e:
+        eprint(
```

Comment:
```suggestion
        _logger.warning(
```

Can we use `_logger.warning` instead?